.github/workflows/ci.yml

```diff
@@ -44,6 +73,9 @@ jobs:
           - isMaster: false
             profile: "singularity"
     steps:
+      - name: install curl
+        run: sudo apt update && sudo apt install -y curl wget
```

Comment:
Do we need wget too, or only curl?

Reply:
oh, I think you're right and that can go.  Thanks!